Alloy/commands/purgetss/index.js

```diff
@@ -0,0 +1,30 @@
+let purgetss = require('purgetss');
+
+module.exports = async function(args, program) {
+	if (args.length === 0) {
+		purgetss.purgeClasses({});
+	} else {
+		args.forEach(command => {
```

Comment:
Do the commands need to be executed in any particular order? Do you need to init before you build?

Reply:
Not at all... It is there.. for people who want to execute several commands at once...

The 'main' command is a one-time execution of `alloy purgetss`.

- From there.. On every compilation it will automatically `parse` all `XML` files y create a new `app.tss` file with only the classes used in `Views`.

- It will automatically re-create the `tailwind.tss` and the `definitions.css` files when there is an update in `config.js`.

Reply:
The `build` command is to re-create `tailwind.tss` without compiling the app... For example to test a new class or a `config.js` change.

The `fonts` command is to copy the icon font libraries... again a one-time command... Unless they deleted the `assets/fonts` folder and want to copy the icon libraries again.

The `build-fonts` command is also a one-time command, unless they added new fonts to `purgetss/fonts` folder and want to re-create the custom `fonts.tss` file with new unicode characters and style rules.